README.md

```diff
@@ -96,6 +108,7 @@ Lychee supports several feature flags:
 
 By default, `native-tls` and `email-check` are enabled.
 
+<a name="features"></a>
```

Comment:
This is automatically done by Github. E.g. https://github.com/lycheeverse/lychee#features